### PR TITLE
feat: add running row Banner

### PR DESCRIPTION
**Pull request by oSlavivna**
I added a banner and its adaptors. 
I used the button component in the banner. 

The banner has a variable with a string intended for future changes to the phrase by the administrator.
image

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import Banner from '@/components/common/Banner/Banner';
```
Comment (biryukov-m) [inline src/app/page.tsx:1]: move Banner component from common to /features/home/sections/BannerSection/index.tsx

File: src/components/common/Banner/Banner.module.scss [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  padding: 8px 120px;
+  background-color: $background-reserved;
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-family: $font-family-base;
+}
+
+.iconBell {
```
Comment (biryukov-m) [inline src/components/common/Banner/Banner.module.scss:16]: Use scss nesting
```
.container {
 ...
  .collection {
       ...
  }
  .iconBell {
  ...  
   }
}
```

etc.

File: src/components/common/Banner/Banner.module.scss [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  font-size: 1.875rem; //30px;
+  font-weight: 600;
+  color: $white-text;
+  white-space: nowrap;
+  display: inline-block;
+  padding-left: 100%;
+  animation: run var(--duration) linear infinite;
+}
+
+.text1 > span {
```
Comment (biryukov-m) [inline src/components/common/Banner/Banner.module.scss:37]: rename text1 to text

Comment (biryukov-m) [inline src/components/common/Banner/Banner.module.scss:37]: use nesting
```
.text {
  & > span {
     ...
  }
}
```

File: src/components/common/Banner/Banner.module.scss [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+@media (max-width: $small-desktop) {
+  .container {
+    padding: 8px 64px;
+  }
+}
+@media (max-width: $tablet) {
+  .container {
+    padding: 8px 40px;
+  }
+  .text1 {
```
Comment (biryukov-m) [inline src/components/common/Banner/Banner.module.scss:64]: nesting

File: src/components/common/Banner/Banner.module.scss [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    font-size: 1.25rem;
+  }
+}
+@media (max-width: $mobile) {
+  .container {
+    padding: 20px;
+    flex-wrap: wrap;
+    max-height: max-content;
+  }
+  .text1 {
```
Comment (biryukov-m) [inline src/components/common/Banner/Banner.module.scss:75]: nesting

File: src/components/common/Banner/Banner.module.scss [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+    font-weight: 600;
+    font-size: 1rem;
+    margin-bottom: 2rem;
+    animation-duration: calc(var(--duration) * 1.8);
+  }
+  .text1 > span {
+    padding-right: 1rem;
+  }
+  .collection {
```
Comment (biryukov-m) [inline src/components/common/Banner/Banner.module.scss:84]: nesting

File: src/components/common/Banner/Banner.tsx [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+          alt="Alert Bell"
+          width={43}
+          height={43}
+        />
+      </div>
+
+      <div className={styles.collection} ref={containerRef}>
+        <div className={styles.text1} ref={textRef}>
+          {Array(5)
+            .fill(null)
```
Comment (biryukov-m) [inline src/components/common/Banner/Banner.tsx:36]: fill with banner var

File: src/components/common/Banner/Banner.tsx [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+            .map((currentValue, index) => (
```
Comment (biryukov-m) [inline src/components/common/Banner/Banner.tsx:37]:
```
(text, index) =>

<span key={index}>{text}</span>
```

**Review by biryukov-m — CHANGES_REQUESTED** [review]
Pull my commits, then:
- Use scss nesting feature among all scss file
- Move banner component to feature/home/bannersection
- Refine map function

**Review by biryukov-m — APPROVED** [review]

**Review by biryukov-m — APPROVED** [review]

**Review by baza-trainee — APPROVED** [review]

**Pull request merged**